src/uk/co/dezzanet/gametable/charactersheet/CharacterSheetPlugin.java

```diff
@@ -0,0 +1,23 @@
+package uk.co.dezzanet.gametable.charactersheet;
+
+import com.galactanet.gametable.GametableFrame;
+import com.galactanet.gametable.IGametablePlugin;
+
+public class CharacterSheetPlugin implements IGametablePlugin {
+
+    private static final String PLUGIN_NAME = "Character sheet";
+    private static final String PANEL_TITLE = "Character Sheet";
```

Comment:
I suppose ideally the plugin would have it's on language text files... but doesn't really matter for now.